central/cve/converter/utils/convert_utils_v2_test.go

```diff
@@ -276,3 +278,9 @@ func getTestComponentID() string {
 
 	return id
 }
+
+func getTestCVEID(testCVE *storage.EmbeddedVulnerability, componentID string) string {
+	id, _ := cve.IDV2(testCVE, componentID)
+
```

Comment:
would we want to turn this into a `mustGetTestCVEID` and add a `require.NoError(err)` here?

Reply:
Moved things around to add `require.NoError(err)`